src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -1312,6 +1312,12 @@ public void setMsqMinSelectableChoices(int qnNumber, int value) {
         executeScript(String.format("$('#%s').val(%d);$('#%s').change();", id, value, id));
     }
 
+    public void fillMsqMinSelectableChoices(int qnNumber, String value) {
+        WebElement inputBox = getMsqMinSelectableChoicesBox(qnNumber);
+        String id = inputBox.getAttribute("id");
+        fillTextBox(browser.driver.findElement(By.id(id)), value);
```

Comment:
i think you can replace `browser.driver.findElement(By.id(id))` with `inputBox`?